### DISCUSSION TITLE
Styling error bars

### DISCUSSION THREAD

**Discussion by sidhenriksen** [sgratzl/chartjs-chart-error-bars#12] [Q&A]
Hey. Love the package. How would I go about styling the error bars? The three main things I'd like to change is 1) line color, 2) cap size, and 3) line width.

**Comment by sgratzl** [comment]
you have the following options to customize their style: 

https://github.com/sgratzl/chartjs-chart-error-bars/blob/36988ae99cc19c814c7fce91bfb40f6ae8148ec0/src/elements/render.ts#L12-L49

**Reply by DigThisData** [reply]
An example would really help.
